src/sentry/api/base.py

```diff
@@ -127,6 +127,12 @@ def load_json_body(self, request):
         except JSONDecodeError:
             return
 
+    def add_metric_tags(self, *args, **kwargs):
```

Comment:
`add_request_metric_tags`? :bike:

Do you need `*args` if you don't use it?

Reply:
Should this be a function that the stats middleware module exports? Right now the logic for these tags is spread across a few different modules.